docs/source/guide/credentials.rst

```diff
@@ -287,6 +288,11 @@ Boto3 will attempt to load credentials from the Boto2 config file. It first chec
     This credential provider is primarily for backwards compatibility purposes with Boto2.
 
 
+Container credential provider
+---------
+You can associate an IAM role with each of your Amazon Elastic Container Service (Amazon ECS) task definitions. Temporary credentials for that role are then available to that task's containers. For more information, see `IAM Roles for Tasks <https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task-iam-roles.html>`_ in the Amazon Elastic Container Service Developer Guide.
```

Comment:
We should probably link to [this page](https://docs.aws.amazon.com/sdkref/latest/guide/feature-container-credentials.html) instead. It has the relevant information to the SDKs and the full scope of this provider, ECS (and IAM Roles for Tasks) is only one aspect of this feature.

Reply:
Rewrote the description to reflect this